docs/DEPLOYMENT_OVERVIEW.md

```diff
@@ -83,6 +83,9 @@ A service deployment lives in a GCP project configured with the items below.
     }
    ]
    ```
+    - The files stored in this bucket are either exported to VWB or downloaded to the user's computer, 
+      depending on the export model configured and selected. If desired they can be set to be automatically deleted
+      as part of the bucket's lifecyle configuration.
```

Comment:
There are other options for export also -- I made that comment in the internal PR based on the export options we have configured for the VWB deployment. Maybe no need to list all the export options here, this is just devops info.

nit: Slightly stronger "recommend setting the bucket to automatically delete objects after some expiration time" rather than "if desired". Also could you please include a link to the GCS lifecycle page? https://cloud.google.com/storage/docs/lifecycle

Reply:
Done, retained the export models phrase for completeness